.github/workflows/test_rocblas.yml

```diff
@@ -66,6 +66,7 @@ jobs:
         run: sudo apt install libgfortran5 -y
 
       - name: Run rocBLAS tests
+        timeout-minutes: 5
```

Comment:
rocblas-test is using gtest/googletest: https://github.com/ROCm/rocBLAS/blob/develop/clients/include/rocblas_test.hpp. Looks like there isn't actually a way to specify a timeout to the test binary though... so I guess this job-level timeout is the best we'll get until that gets refactored somehow.